lib/sync-settings.coffee

```diff
@@ -1,9 +1,8 @@
 # imports
 {BufferedProcess} = require 'atom'
-GitHubApi = require 'github'
-_ = require 'underscore-plus'
-PackageManager = require './package-manager'
 fs = require 'fs'
+_ = require 'underscore-plus'
+[GitHubApi, PackageManager] = []
```

Comment:
I meant to add it in the code, as after the pull is merged it won't be so obvious why the imports are moved